modules/tensor_mechanics/src/materials/GeneralizedMaxwellModel.C

```diff
@@ -71,6 +71,8 @@ GeneralizedMaxwellModel::GeneralizedMaxwellModel(const InputParameters & paramet
 
   _components = _eta_i.size();
   _has_longterm_dashpot = (_eta_i.size() == _Ci.size() + 1);
+
+  issueGuarantee(_elasticity_tensor_name, Guarantee::ISOTROPIC);
```

Comment:
Would it be better to do this in a base class?  I'm OK either way -- just asking.  Maybe in general it's possible to have an anisotropic version of one of these models, in which case it makes the most sense here.

Reply:
We can have, in theory, an anisotropic version of these models. I'm not sure anyone would use that in practice, so we could force the guarantee in the base class, and let derived classes revoke that guarantee on a case-by-case basis?

Reply:
I'd say let's just keep it as you have it and guarantee that at the derived class level then.

Reply:
Works for me! Actually, it's probably wiser to not guarantee something in the base class, to avoid guaranteeing something to be isotropic when it is not...